backend/api/EventHandlers/MqttEventHandler.cs

```diff
@@ -206,7 +206,10 @@ private async void OnMissionUpdate(object? sender, MqttReceivedArgs mqttArgs)
                 return;
             }
 
-            var flotillaMissionRun = await missionRunService.UpdateMissionRunStatusByIsarMissionId(isarMission.MissionId, status);
+            MissionRun? flotillaMissionRun;
+            try { flotillaMissionRun = await missionRunService.UpdateMissionRunStatusByIsarMissionId(isarMission.MissionId, status); }
+            catch (MissionRunNotFoundException) { return; }
+
             if (flotillaMissionRun is null)
```

Comment:
Shouldn't this logging be inside the above catch block, in case no mission was found?

Reply:
Instead I removed the nullable on mission run here and changed the function as it can no longer return null. This means we could remove the check altogether.